chain/beacon.go

```diff
@@ -90,83 +88,14 @@ func Verify(pubkey kyber.Point, prevSig, signature []byte, round uint64) error {
 func Message(currRound uint64, prevSig []byte) []byte {
 	h := sha256.New()
 	h.Write(prevSig)
-	h.Write(roundToBytes(currRound))
+	h.Write(RoundToBytes(currRound))
 	return h.Sum(nil)
 }
 
-// TimeOfRound is returning the time the current round should happen
-func TimeOfRound(period time.Duration, genesis int64, round uint64) int64 {
-	if round == 0 {
-		return genesis
+func shortSigStr(sig []byte) string {
+	max := 3
+	if len(sig) < max {
```

Comment:
It's just a debug/logging feature to allow printing the first characters instead of the fully long string, it looks much better when looking at the log :D